src/resources/MachineLearning/CaseClassificationConfiguration/CaseClassificationConfigurationInterfaces.ts

```diff
@@ -157,3 +157,52 @@ export interface CaseClassificationContentFields {
     fields: CaseClassificationContentField[];
     query: string;
 }
+
+interface CaseClassificationFieldCountAdvancedMode {
+    /**
+     * The query to use for model building.
+     */
+    advancedQuery: string;
+}
+
+interface CaseClassificationFieldCountStandardMode {
+    /**
+     * The names of the sources containing the cases to use for model building.
+     */
+    sources: string[];
+    /**
+     * An array of filtering conditions.
+     */
+    filterConditions?: FilterConditions[];
+    /**
+     * The field value to use as language.
+     */
+    languageField: string;
+    /**
+     * The time period for which to extract cases for model building.
+     * Must contain an export period or a start time and end time.
```

Comment:
we don't need this anymore as we're removing the custom start and end date selection option.

Reply:
i've removed the second line, we must still provide it for the `/params` call, but it will be hardcoded in the call instead of being customizable like you said